src/lib.rs

```diff
@@ -525,10 +510,45 @@ where
         }
     }
 
-    while let Some(msg) = b.next() {
+    if let Some(log) = cfg.logger.as_ref() {
+        info!(log, "compiled all datapath programs, ccp ready");
+    }
+
+    while let Some((msg, recv_addr)) = b.next() {
         match msg {
+            Msg::Rdy(r) => {
+                if dp_to_flowmap.remove(&recv_addr).is_some() {
```

Comment:
is there a difference in the two branches of the if other than what gets logged?

Reply:
no